newIDE/app/src/fixtures/GDevelopServicesTestData.js

```diff
@@ -19,6 +19,13 @@ import {
 } from '../Utils/GDevelopServices/Asset';
 
 export const indieFirebaseUser: FirebaseUser = {
+  uid: 'indie-user',
+  providerId: 'fake-provider.com',
+  email: 'indie-user@example.com',
+  emailVerified: false,
+};
+
+export const indieFirebaseVerifiedUser: FirebaseUser = {
```

Comment:
ubernitpicking (sorry): it may be a question of style, but I would name this... `indieVerifiedFirebaseUser` 😁
Because this thing IS a "FirebaseUser", because it has the type "FirebaseUser". So this means Firebase can't be removed from User, they must stay together. So the adjective "Indie" and "Verified" comes before: verifiedIndieFirebaseUser or indieVerifiedFirebaseUser.

Sorry 😅

Reply:
Good feedback :)